txcache/selectionTracker_test.go

```diff
@@ -10,27 +10,10 @@ import (
 	"github.com/multiversx/mx-chain-core-go/core"
 	"github.com/multiversx/mx-chain-core-go/data/block"
 	"github.com/multiversx/mx-chain-go/common/holders"
-	"github.com/multiversx/mx-chain-go/state"
-	testscommonState "github.com/multiversx/mx-chain-go/testscommon/state"
 	"github.com/multiversx/mx-chain-go/testscommon/txcachemocks"
 	"github.com/stretchr/testify/require"
 )
 
-func getSelectionSessionMock() SelectionSession {
```

Comment:
Unfortunate function. Occupied a "good" and "generic" name in the global package namespaces, but was only used in a specific test.